cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -64,6 +64,8 @@ API Changes
 Known Issues
 ------------
 
+- CDAP has been tested on and supports CDH 4.2.x through CDH 5.3.x. It has not been tested on later versions.
```

Comment:
I don't think we've tested Hadoop 2.7 or Zookeeper 3.4.6 at this point, as they are not yet in any of the major distributions.  Agreed on the Hive version suggestion.